app/lib/report_helper.rb

```diff
@@ -20,7 +20,12 @@ def report_details(report)
   end
 
   def compute_taxon_zscores(report)
-    summary = report.background.summarize
+    bg = report.background
```

Comment:
You'll need to `pull --rebase origin master` and do a manual merge/resolve here.    Otherwise, as is, this pull request would revert some recent changes :)

Reply:
Hmmm, not exactly.   It's very puzzling why github is showing the diff against an older master.